uaclient/cli.py

```diff
@@ -1865,6 +1865,21 @@ def _warn_about_new_version(cmd_args=None) -> None:
         logging.warning(NEW_VERSION_NOTICE.format(version=new_version))
 
 
+def _warn_about_output_redirection(cmd_args) -> None:
+    """Warn users that the user readable output may change."""
+    if (
+        cmd_args.command in ("status", "security-status")
+        and not sys.stdout.isatty()
```

Comment:
Sorry, what is the idea behind the extra check ?

Reply:
We are only printing the warning to stderr if the stdout is not a tty - if the caller is piping it for instance. If just called in the terminal, the commands should not print any warning.

Reply:
This mimics what apt does btw
`apt update` -> no warning
`apt update > output_text` -> warning
`apt update | cat` -> warning